packages/react-components/react-teaching-popover/src/components/TeachingPopoverBody/useTeachingPopoverBodyStyles.styles.ts

```diff
@@ -11,28 +11,28 @@ export const useMediaStyles = makeStyles({
   base: {
     ...shorthands.gridArea('media'),
     ...shorthands.overflow('hidden'),
-    width: '288px',
+    width: '286px',
```

Comment:
We should avoid changing the size as the popover body is required to be 288px width as spec'd in design, curious why the width would affect the arrow here as it should be capable of dynamic sizing.